File: MODUL-6/SOAL 1/PRAK601-2410817120017-RIKAFAULIANARAHMI.py
baris, kolom = map(int, input().split())

angka = list(map(int, input().split()))

matriks = [[0 for _ in range(kolom)] for _ in range(baris)]

index = 0
for i in range(baris):
    for j in range(kolom):
        matriks[i][j] = angka[index]
        index += 1

for i in range(baris):
    for j in range(kolom):
        print(matriks[i][j], end=" ")
    print() 

File: MODUL-6/SOAL 1/PRAK601-2410817120017-RIKAFAULIANARAHMI.c
#include <stdio.h>

int main() {
  int baris, kolom;
  scanf("%d %d", &baris, &kolom);

  int data[baris][kolom];
  for (int i = 0; i < baris; i++) {
    for (int j = 0; j < kolom; j++) {
      scanf("%d", &data[i][j]);
    }
  }

  for (int i = 0; i < baris; i++) {
    for (int j = 0; j < kolom; j++) {
      printf("%d ", data[i][j]);
    }
    printf("\n");
  }

  return 0;
}
